plugin/trino-hive/src/main/java/io/trino/plugin/hive/HiveConfig.java

```diff
@@ -145,6 +146,7 @@
     private boolean optimizeSymlinkListing = true;
 
     private boolean legacyHiveViewTranslation;
+    private DataSize targetMaxFileSize = DataSize.of(1, GIGABYTE);
```

Comment:
I am a bit curious where did the default value come from. But 1GB seems a safe default.

Reply:
It is arbitrary.  It makes for pretty large files by default... generally it is large enough that most things aren't going to hit it.  We can easily change this in the future.